modin/pandas/dataframe.py

```diff
@@ -112,6 +112,8 @@ class DataFrame(BasePandasDataset):
     ``pd.read_csv``).
     """
 
+    _AXIS_ORDERS = ["index", "columns"]
+    _AXIS_LEN = len(_AXIS_ORDERS)
```

Comment:
```suggestion
    _AXES = ["index", "columns"]
    _NDIM = len(_AXIS_ORDERS)
```

Why do we need these new attributes?